server/models/User.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+
+const UserSchema = mongoose.Schema({
+	firstname: {
+		type: String,
+		required: true,
+	},
+	lastname: {
+		type: String,
+		required: true,
+	},
+	email: {
+		type: String,
+		required: true,
+		unique: true,
+	},
+	password: {
+		type: String,
+		required: true,
+	},
+	bio: {
+		type: String,
+	},
+	friends: {
+		type: Array,
+	},
+	notifications: {
+		type: Array,
+	},
+	receivedfriendrequests: {
+		type: Array,
+	},
+	sentfriendrequests: {
+		type: Array,
+	},
+	post: {
+		type: Array,
+	},
+	comments: {
+		type: Array,
+	},
```

Comment:
No change needed. But perhaps these would be better to commit when adding each of these other tables.

It makes the commits easier to read as all the table to table links are included in the same commit/RP.

Right now these are a little strange to read as they do nothing (yet).